src/backport.ts

```diff
@@ -303,7 +313,7 @@ const backport = async ({
 
     const title = getTitle({ base, number, title: originalTitle });
     const merged_by = originalMergedBy?.login ?? "";
-    const runUrl = `${serverUrl}/${owner}/${repo}/actions/runs/${runId}/jobs/${runNumber}`;
+    const runUrl = `${serverUrl}/${owner}/${repo}/actions/runs/${runId}`;
```

Comment:
this link is broken

e.g., https://github.com/sourcegraph/sourcegraph/actions/runs/6123459521/jobs/15021

the job id is not exposed from the context, let's just go as far as the summary page of the workflow run

it's good enough